zerver/views/streams.py

```diff
@@ -597,13 +597,15 @@ def get_subscribers_backend(request: HttpRequest, user_profile: UserProfile,
 def get_streams_backend(
         request: HttpRequest, user_profile: UserProfile,
         include_public: bool=REQ(validator=check_bool, default=True),
+        include_web_public: bool=REQ(validator=check_bool, default=True),
```

Comment:
I think this should perhaps default to `False`; they should already be included through `include_public`, and I don't want to make it more complicated to ask for "just my subscriptions" (as this change visibly does in the API documentation above).